Website/src/rendering/src/components/Navigation/MainNavigation.tsx

```diff
@@ -73,11 +74,13 @@ const MainNavigation = (props: MainNavigationProps): JSX.Element => {
                 <a>About Us</a>
               </Link>
             </li>
-            <li className="text-menu-item">
-              <Link href="/shop">
-                <a>Shop</a>
-              </Link>
-            </li>
+            {isCommerceEnabled ? (
```

Comment:
We try to avoid logical conditions in the JSX return statement to ensure easier code comprehension. We generally extract conditional parts in consts:

```typescript
const shopLink = isCommerceEnabled && (
  <li className="text-menu-item">
    <Link href="/shop">
      <a>Shop</a>
    </Link>
  </li>
);
```

And we then use it in JSX with `{shopLink}`